symfony/plugins/orangehrmMarketPlacePlugin/lib/dao/MarketplaceDao.php

```diff
@@ -91,7 +91,7 @@ public function uninstallAddon($addonId)
             return $q->execute();
             // @codeCoverageIgnoreStart
         } catch (Exception $e) {
-            throw new DaoException($e->getMessage(), $e->getCode(), $e);
+            throw $e;
```

Comment:
why catch the exception if you throw same ?

Reply:
corrected.